package.json

```diff
@@ -18,7 +19,11 @@
     "react-native-web": "0.17.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.9"
+    "@babel/core": "^7.12.9",
+    "jest": "^27.4.7"
+  },
+  "jest": {
+    "preset": "react-native"
```

Comment:
Since we are using Expo framework, I would suggest to set "preset" value to "jest-expo", just to keep things consistent. 

Check out here https://docs.expo.dev/guides/testing-with-jest/